Add example for configuring JS collocation in _Layout.cshtml (#33190)
dotnet/AspNetCore.Docs: aspnetcore/includes/js-collocation.md

  }
  ```

> The default layout `(Pages/Shared/_Layout.cshtml)` can be configured to include collocated JS files, eliminating the need to configure each page individually:

Reviewer: ```suggestion
The default layout `Pages/Shared/_Layout.cshtml` can be configured to include collocated JS files, eliminating the need to configure each page individually:
```

>   You can configure the default layout `(Pages/Shared/_Layout.cshtml)` to always include collocated JS files, eliminating the need to configure each page individually:

Reviewer: ```suggestion
The default layout `(Pages/Shared/_Layout.cshtml)` can be configured to include collocated JS files, eliminating the need to configure each page individually:
```
